Gemfile

```diff
@@ -14,7 +14,9 @@ gem 'uglifier', '>= 1.3.0'
 gem 'coffee-rails', '~> 4.2'
 gem 'turbolinks', '~> 5'
 gem 'jbuilder', '~> 2.5'
-
+gem 'awesome_print'
```

Comment:
Should probably move this to `:development, :test` group

Reply:
okidoki!